### PR TITLE
add recruitment link to wrapped emails

### PR DESCRIPTION
**Pull request by groovecoder**
This branch is already deployed on dev.fxprivaterelay.nonprod.cloudops.mozgcp.net if you want to spot-check it there.

### CODE REVIEW

File: privaterelay/settings.py [hunk 1]
```diff
@@ -109,6 +109,7 @@
 GOOGLE_ANALYTICS_ID = config('GOOGLE_ANALYTICS_ID', None)
 RECRUITMENT_BANNER_LINK = config('RECRUITMENT_BANNER_LINK', None)
 RECRUITMENT_BANNER_TEXT = config('RECRUITMENT_BANNER_TEXT', None)
```
Comment (say-yawn) [inline privaterelay/settings.py:111]:
```suggestion
RECRUITMENT_BANNER_TEXT = config('RECRUITMENT_BANNER_TEXT', None)
RECRUITMENT_EMAIL_BANNER_TEXT = config('RECRUITMENT_EMAIL_BANNER_TEXT', None)
```
I assume the `RECRUITMENT_BANNER_TEXT` config was for banner on the dashboard and we probably want a separate banner text like you did for the link?

File: emails/views.py [hunk 2]
```diff
@@ -330,7 +330,9 @@ def _sns_message(message_json):
             'display_email': display_email,
             'SITE_ORIGIN': settings.SITE_ORIGIN,
             'has_attachment': bool(attachments),
-            'faq_page': settings.SITE_ORIGIN + reverse('faq')
+            'faq_page': settings.SITE_ORIGIN + reverse('faq'),
+            'survey_text': settings.RECRUITMENT_BANNER_TEXT,
```
Comment (say-yawn) [inline emails/views.py:334]:
```suggestion
            'survey_text': settings.RECRUITMENT_EMAIL_BANNER_TEXT,
```

**Review by say-yawn — CHANGES_REQUESTED** [review]
Code LGTM. Minor suggestions on creating another config for email banner text.

**Comment by groovecoder** [comment]
Committed suggestions and tested on dev server. Merging.

**Pull request merged**